controllers/managedocs_controller.go

```diff
@@ -1669,6 +1680,16 @@ func (r *ManagedOCSReconciler) findOCSVolumeClaims() (bool, error) {
 	return false, nil
 }
 
+func (r *ManagedOCSReconciler) findStorageConsumerList() (bool, error) {
```

Comment:
This function returns a bool so the name is a little bit misleading. Can we please rename it to `hasOCSStorageConsumers`? In in the same manner rename `findOCSVolumeClaims` to `hasOCSVolumes`? 

Please note that `findOCSVolumeClaims` is not searching for volume claims, it searches for volumes, so the current name is also problematic in that regard.

Why the need to mention List in the function name? can we follow the `findOCSVolumeClaims` convention and call this `findOCSStorageConsumers`. Also `findOCSVolumeClaims` does not search for VolumeClaims, it searches for Volumes so let's also rename it to `